docs/en/latest/admin-api.md

```diff
@@ -88,6 +88,7 @@ Note: When the `Admin API` is enabled, it will occupy the API prefixed with `/ap
 | script           | False                                    | Script      | See [Script](architecture-design/script.md) for more                                                                                                                                                                                                                                                                                                                                                                               |                                                      |
 | upstream         | False                                    | Upstream    | Enabled Upstream configuration, see [Upstream](architecture-design/upstream.md) for more                                                                                                                                                                                                                                                                                                                                           |                                                      |
 | upstream_id      | False                                    | Upstream    | Enabled upstream id, see [Upstream](architecture-design/upstream.md) for more                                                                                                                                                                                                                                                                                                                                                      |                                                      |
+| upstream_timeout | False                                    | Upstream    | Set the upstream timeout for connection, sending and receiving messages of the route. This option will overwrite the [timeout](#upstream) option which set in upstream configuration.                                                                                                                                                                                                                                                                                                         | {"connect": 3, "send": 3, "read": 3}              |
```

Comment:
ditto and some other places. Also the type maybe should be `Auxiliary`? I'm not sure about this